vars/buildPlugin.groovy

```diff
@@ -12,6 +12,8 @@ def call(Map params = [:]) {
   def failFast = params.containsKey('failFast') ? params.failFast : true
   def timeoutValue = params.containsKey('timeout') ? params.timeout : 60
   def gitDefaultBranch = params.containsKey('gitDefaultBranch') ? params.gitDefaultBranch : null
+  def artifactCachingProxyEnabled = params.containsKey('artifactCachingProxyEnabled') ? params.artifactCachingProxyEnabled : false
```

Comment:
The default value to `false` does not map the documentation (https://github.com/jenkins-infra/pipeline-library/pull/458/files#diff-1194561cbea346acb1bea37d81fdf5c70def982dcbb0c64eca2a96bc51941124R43). I assume it's temporary as the PR is work in progress, but adding a comment to avoid forgetting before merging

Reply:
I think the artifact caching proxy should be opt-in until we validate it works correctly.